Feature: Show pull request CI actions (#19)
andrecavallari/rails-sample: README.md

This is a sample rails app for showing my skills with ruby on rails

> And this is just a comment in this pull request to show CI actions

Author: ```suggestion
Just a comment to show how this pull request works
```
Maybe this sounds better?
